data/config.py

```diff
@@ -303,7 +303,9 @@ def print(self):
 
 
 # ----------------------- MASK BRANCH TYPES ----------------------- #
-
+#FIXME cannot import mask_type ?
+mask_type_DIRECT = 0
```

Comment:
FIXME: I can import `from data.config import mask_type` but then I cannot acces `mask_type.direct`, so this is a workaround, do you know a better way, please?

Reply:
see https://github.com/dbolya/yolact/pull/323/files#diff-2f8aa115c021e3cda66495710492905dR69

Reply:
I ran this branch locally and replacing mask_type_DIRECT with mask_type.direct, etc. worked just fine. Are you sure there's an issue here? Most intellesense won't show mask_type.direct because it's a config object (that converts a dictionary to a python object).

Reply:
Fixed in 
cf2d4e6 , thanks for pointing out!